foundation/foundation-archetype/src/test/resources/test-project-archetype-script/maven-build-cache-config.xml

```diff
@@ -9,22 +9,11 @@
   #L%
   -->
 
-<!---
- Licensed to the Apache Software Foundation (ASF) under one or more
- contributor license agreements.  See the NOTICE file distributed with
- this work for additional information regarding copyright ownership.
- The ASF licenses this file to You under the Apache License, Version 2.0
- (the "License"); you may not use this file except in compliance with
- the License.  You may obtain a copy of the License at
-
-      http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
--->
+<!--
```

Comment:
I: what this comment says...

Reply:
A: Typo in "generate this *inside* the archetype"

Reply:
✅ Good catch, I updated to now read:

> This file should not be here - we generate this inside the archetype, so it is unclear why we need it
copied over the one we generate. We need to investigate more to determine how to cull this b/c having
two that compete needs to be clarified.